Sources/FaunaDB/Query.swift

```diff
@@ -1159,6 +1285,12 @@ public struct Database: Fn {
         self.call = fn("database" => name)
     }
 
+    /// - Parameter name: The database name.
+    /// - Parameter scope: The scope database.
+    public init(_ name: String, scope: Expr) {
```

Comment:
You can add `scope` as a optional parameter to the original constructor. No need to overload.
As a side note, `fn` already ignores nil values.